pkg/registry/registrytest/etcd.go

```diff
@@ -222,6 +222,9 @@ func (t *Tester) emitObject(obj runtime.Object, action string) error {
 			return err
 		}
 		_, _, err = t.storage.Delete(ctx, accessor.GetName(), nil)
+		if err != nil {
```

Comment:
Instead of special casing this error, rename the `err` from `Accessor` to `errAccessor` so it does not shadow the error we care about.